pkg/osutil/osutil.go

```diff
@@ -65,9 +65,9 @@ func Run(timeout time.Duration, cmd *exec.Cmd) ([]byte, error) {
 	err := cmd.Wait()
 	close(done)
 	if err != nil {
-		text := fmt.Sprintf("failed to run %q: %v", cmd.Args, err)
+		text := fmt.Sprintf("failed to run '%q': %v", strings.Join(cmd.Args, " "), err)
```

Comment:
This is worse. It does not allow to understand number of arguments and exact arguments passed,